torch_np/_helpers.py

```diff
@@ -70,7 +81,7 @@ def result_or_out(result_tensor, out_array=None, promote_scalar=False):
         out_tensor.copy_(result_tensor)
         return out_array
     else:
-        return asarray(result_tensor)
+        return array_from(result_tensor)
```

Comment:
Why this change?

Reply:
Purely technical. Not a big deal in this file (just makes it easier to grep / more uniform). 
The general direction is centralizing imports from `_ndarray.py` to minimize/centralize local imports.

Reply:
makes sense, and thank you for the context!